indexer/packages/postgres/src/stores/token-table.ts

```diff
@@ -0,0 +1,132 @@
+import { DateTime } from 'luxon';
```

Comment:
`token-table` is a little vague. Can we be a bit more specific? perhaps `firebase-notification-token-table`?